### ISSUE TITLE
how to crop gif?

### ISSUE THREAD

**Issue by crazyjin** [rainycape/magick#30]
i use func crop and then i got result looks like: 
![wkhkafqfotmab_rjaaaqrv3jxhu857_c_100_100_100_100](https://cloud.githubusercontent.com/assets/3919337/5576553/14ec12ce-9039-11e4-8320-b73a6261333c.GIF)

**Comment by xealot** [comment]
Yeah, I have this issue also. The image needs to be repaged, on the command line it would be `+repage` but I'm not sure how to do it with the API provided.
